opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -48,9 +48,7 @@ class BaseBoundInstrument:
         metric: The metric that created this bound instrument.
     """
 
-    def __init__(
-        self, labels: Tuple[Tuple[str, str]], metric: metrics_api.MetricT
-    ):
+    def __init__(self, labels: Tuple[Tuple[str, str]], metric: "MetricT"):
```

Comment:
I think this should be promoted to the class level generic `Generic[MetricT]`. Understand if you'd rather do in a separate PR tho